scripts/install-chromedriver.mjs

```diff
@@ -0,0 +1,70 @@
+import _ from 'lodash';
+import { ChromedriverStorageClient } from 'appium-chromedriver';
+import axios from 'axios';
+
+const VERSION_LATEST = 'LATEST';
+
+function getChromedriverUrl() {
+    return process.env.CHROMELABS_URL || 'https://googlechromelabs.github.io'
+}
+
+function getChromedriverVersion() {
+    return process.env.npm_config_chromedriver_version ||
+        process.env.CHROMEDRIVER_VERSION ||
+        VERSION_LATEST;
+}
+
+async function formatCdVersion (ver) {
```

Comment:
why cannot we import these function instead of duplicating them?

Reply:
same above

Reply:
Do you mean exporting them from `appium-chromedriver`'s index and import them from the module? No special reason but just to not modify the chromedriver side only for these usage in scripts. e.g. Move formatCdVersion from test to somewhere and exporting it.
Or importing them from `appium-chromedriver/build/lib/constants`...? Then, appium-chromedriver may not know if the path could be used by other modules (this package.) so probably not good to do it

Reply:
there is no need to export from index. You may export directly from build/lib. See, for example, https://github.com/appium/appium-windows-driver/blob/dba67a91039f8919373e7e913f862c2b8dcda867/scripts/install-wad.mjs#L7

Reply:
Yea, then you mean `import LATEST from 'appium-chromedriver/build/lib/constants';` correct? Then, the https://github.com/appium/appium-chromedriver should keep the structure to not break https://github.com/appium/appium-chromium-driver (this module). I wanted to avoid calling other package's internal path such as `build/lib...` since it means https://github.com/appium/appium-chromedriver maintenance should keep taking care of https://github.com/appium/appium-chromium-driver

Reply:
you may still call the script directly from node_modules in appium-chromium-driver. Although, I would consider either adding a separate script to the latter then. Same with uia2 driver